public/js/p3/widget/CircularViewerContainer.js

```diff
@@ -161,8 +251,14 @@ define([
 			}, "outer", true);
 
 			//this.addFeatureTrack("CDS - FWD", "CDS forward strand", this.state.genome_ids[0], "and(eq(annotation,PATRIC),eq(feature_type,CDS))", null, "blue", null);
-			this.addFeatureTrack("CDS - FWD", "CDS forward strand", this.state.genome_ids[0], "and(eq(annotation,PATRIC),eq(feature_type,CDS),eq(strand,%22\+%22))", true, "#307D32", null)
-			this.addFeatureTrack("CDS - REV", "CDS reverse strand", this.state.genome_ids[0], "and(eq(annotation,PATRIC),eq(feature_type,CDS),eq(strand,%22-%22))", false, "#833B76", null)
+			this.addFeatureTrack("CDS - FWD", "CDS forward strand", this.state.genome_ids[0], "and(eq(annotation,PATRIC),eq(feature_type,CDS),eq(strand,%22\+%22))", true, "#307D32", null);
+			this.addFeatureTrack("CDS - REV", "CDS reverse strand", this.state.genome_ids[0], "and(eq(annotation,PATRIC),eq(feature_type,CDS),eq(strand,%22-%22))", false, "#833B76", null);
+			this.addSpGeneTrack("AMR Genes", "AMR Genes", this.state.genome_ids[0], "&in(property,(%22Antibiotic%20Resistance%22))", null, "red", null);
```

Comment:
isn't this equivalent to 
```eq(property,Antibiotic Resistance)```

Reply:
Again, this was used for multiple properties before

Reply:
I can change this